data-mermaid/mermaid-webapp
# feat: [M467] implement bleaching summary stats on collect record page…

… including refactoring submitted page to reuse some of the code. Implemented forgotten percent cover observations table for submitted page.

I stole some code from the bleaching submitted page. Renamed some of the props and files. The diff doesnt show up as renames, but rather new and deleted files which is annoying.


Steps to test:

- Add some observations to the bleaching collect record page and make sure the summary stats make sense
- Do same for submitted page.
